apps/dataset-browser/e2e/content-pages.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {expect, test} from '@playwright/test';
+
+test.describe('Content pages', () => {
+  test('loads the contact page', async ({page}) => {
```

Comment:
Does Playwright have an `it()` function (since our other tests following that naming convention)?